query/branch/default_branch_requires_signed_commits.sql

```diff
@@ -0,0 +1,13 @@
+select
+  -- Required Columns
+  r.full_name as resource,
+  case
+    when b.signatures_protected_branch_enabled is not null and (b.signatures_protected_branch_enabled)::bool = true then 'ok'
+    else 'alarm'
+    end as status,
+    r.full_name || ' default branch ' || r.default_branch || case when(b.signatures_protected_branch_enabled is not null and (b.signatures_protected_branch_enabled)::bool = true) then ' requires ' else ' does not require ' end || 'signed commits before merge.' as reason,
```

Comment:
can we update the format as per - https://github.com/turbot/steampipe-mod-github-compliance/pull/4/files#r946451106